frontend/src/components/SearchBar.js

```diff
@@ -35,7 +35,7 @@ class SearchBar extends Component {
         this.props.searchCallback(results);
       }
     } else {
-      console.log(message);
+      this.props.alertCallback(message);
```

Comment:
nice work with the callbacks hopefully didn't get too annoying to deal with :face_with_head_bandage: